dashboard/app/models/learning_goal_ai_evaluation.rb

```diff
@@ -2,40 +2,38 @@
 #
 # Table name: learning_goal_ai_evaluations
 #
-#  id               :bigint           not null, primary key
-#  user_id          :integer
-#  learning_goal_id :integer
-#  project_id       :integer
-#  project_version  :string(255)
-#  understanding    :integer
-#  created_at       :datetime         not null
-#  updated_at       :datetime         not null
-#  requester_id     :integer
-#  ai_confidence    :integer
-#  status           :integer          default(0)
+#  id                      :bigint           not null, primary key
+#  rubric_ai_evaluation_id :bigint           not null
+#  learning_goal_id        :bigint           not null
+#  understanding           :integer
+#  ai_confidence           :integer
+#  created_at              :datetime         not null
+#  updated_at              :datetime         not null
 #
 # Indexes
 #
-#  index_learning_goal_ai_evaluations_on_learning_goal_id  (learning_goal_id)
-#  index_learning_goal_ai_evaluations_on_requester_id      (requester_id)
-#  index_learning_goal_ai_evaluations_on_user_id           (user_id)
+#  index_learning_goal_ai_evaluations_on_learning_goal_id         (learning_goal_id)
+#  index_learning_goal_ai_evaluations_on_rubric_ai_evaluation_id  (rubric_ai_evaluation_id)
```

Comment:
after adding rubric_id to rubric_ai_evaluations, I do have a concern that we could get into a bad state where:
```
learning_goal_ai_evaluation.rubric_ai_evaluation.rubric != learning_goal_ai_evaluation.learning_goal.rubric
```
since rubric and learning_goal do not change very often, and LGAIs are going to be created after RAIs, my thought would be to solve this by adding a validation here asserting that the two values in the above inequality are in fact equal (and not nil).

Reply:
I still think this validation would be worth adding. I have an idea for how to do it, and would be happy to take this as a follow-up

Reply:
I can still add this validation. It would be in the LearningGoalAiEvaluation table and validate that the `rubric_id` of both the RAI and the LG match.